sql-plugin/src/main/scala/org/apache/spark/sql/rapids/datetimeExpressions.scala

```diff
@@ -1132,6 +1132,65 @@ case class GpuFromUTCTimestamp(
   }
 }
 
+class ToUTCTimestampExprMeta(
+    expr: ToUTCTimestamp,
+    override val conf: RapidsConf,
+    override val parent: Option[RapidsMeta[_, _, _]],
+    rule: DataFromReplacementRule)
+  extends BinaryExprMeta[ToUTCTimestamp](expr, conf, parent, rule) {
+
+  private[this] var timezoneId: ZoneId = null
+
+  override def tagExprForGpu(): Unit = {
```

Comment:
It looks like `ToUTCTimestampExprMeta` has basically the same logic for whether it will run on the GPU as `FromUTCTimestampExprMeta`. I think these 2 classes should be refactored to share this logic

Reply:
Done.